src/spyglass/lfp/analysis/v1/lfp_band.py

```diff
@@ -363,14 +359,9 @@ def make(self, key: dict) -> None:
         timestamps = np.asarray(lfp_object.timestamps)
         # get the indices of the first timestamp and the last timestamp that
         # are within the valid times
-        included_indices = interval_list_contains_ind(
-            lfp_band_valid_times, timestamps
-        )
-        # pad the indices by 1 on each side to avoid message in filter_data
-        if included_indices[0] > 0:
-            included_indices[0] -= 1
-        if included_indices[-1] != len(timestamps) - 1:
-            included_indices[-1] += 1
+        included_indices = lfp_band_valid_times.contains(
```

Comment:
Does the padding belong in Interval?

Reply:
It is the case that this is the only place this is used. I chose to incorporate it so the full operation would be contained to the contains method, opening the door to other pipelines using the logic